src/cache_cf.cc

```diff
@@ -2870,22 +2846,12 @@ parse_refreshpattern(RefreshPattern ** head)
                   ) {
             debugs(22, DBG_PARSE_NOTE(2), "UPGRADE: refresh_pattern option '" << token << "' is obsolete. Remove it.");
         } else
-            debugs(22, DBG_CRITICAL, "ERROR: refreshAddToList: Unknown option '" << pattern << "': " << token);
-    }
-
-    if ((errcode = regcomp(&comp, pattern, flags)) != 0) {
-        char errbuf[256];
-        regerror(errcode, &comp, errbuf, sizeof errbuf);
-        debugs(22, DBG_CRITICAL, "" << cfg_filename << " line " << config_lineno << ": " << config_input_line);
-        debugs(22, DBG_CRITICAL, "ERROR: refreshAddToList: Invalid regular expression '" << pattern << "': " << errbuf);
-        xfree(pattern);
-        return;
+            debugs(22, DBG_CRITICAL, "ERROR: Unknown refresh_pattern option: " << token);
```

Comment:
> We are going with the non-sentence casing now right?

Not for level-0/1 messages, especially those that start with an all-caps label followed by a colon. Should I try to summarize the few existing rules so that we can polish them together (outside of this PR)?

I believe the casing in PR is better because it matches many American English [styles](https://www.tckpublishing.com/capitalization-after-colons/), and we try to follow American English variants rather than the British ones.